김보민/RestApiStudy/src/main/java/net/skhu/restapistudy/domain/Movie.java

```diff
@@ -0,0 +1,24 @@
+package net.skhu.restapistudy.domain;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Movie {
+    private Long id;
+    private String name;
+    private String genre;
+    private Long count;
+
+    @Builder
+    public Movie(Long id, String name, String genre, Long count){
+        this.id = id;
+        this.name=name;
+        this.genre=genre;
+        this.count=count;
+    }
+    public void initId(Long id)
+    {this.id=id;}
```

Comment:
```suggestion
    public void initId(Long id) {
        this.id=id;
    }
```
이렇게 바꾸면 더 가독성이 좋을 거 같아요~